src/components/posts/uploadRealmoji.vue

```diff
@@ -138,66 +108,148 @@ export default {
           console.log(data);
           this.$store.dispatch("getPosts").then((d) => {
             this.loading = false;
-            this.file = null;
-            this.imageurl = null;
           });
         })
         .catch((err) => {
           console.log(err);
           this.loading = false;
           this.$store.commit("error", "Failed to upload");
         });
+      } else {
+        this.reactRealmoji(this.emojis["⚡"])
+      } 
+      
+    },
+    async reactRealmoji(realmoji) {
+      
+      if (realmoji.emoji == "⚡") {
+        fetch(`${this.$store.state.proxyUrl}/https://mobile.bereal.com/api/content/realmojis/instant?postId=${this.postID}&postUserId=${this.user.id}`, {
```

Comment:
we should extract all the api calls to a seperate file, but let's maybe keep it as is for now